### DISCUSSION TITLE
Should we remove ArgoCD's own UI from ks-devops?

### DISCUSSION THREAD

**Discussion by chengleqi** [kubesphere/ks-devops#656] [General]
Looks like we don't need ArgoCD's own UI

1. We just need the controller provided by ArgoCD.
2. The Synchronization is one way ( gitops.kubesphere.io/Application -> argoproj.io/Application ). When we open the ArgoCD's own UI and create a argoproj.io/Application, it won't appear in kubesphere UI, it' ll cause inconsistent issue.

**Comment by LinuxSuRen** [comment]
Yes. You're right. We only take Argo CD as the backend. Please feel free to help to improve it if you are interested in it.

**Reply by chengleqi** [reply]
I’ll try😃

**Reply by LinuxSuRen** [reply]
Please let us know if you need any help.
